### ISSUE TITLE
Travis failing on older Pythons

### ISSUE THREAD

**Issue by mpirnat** [gregmalcolm/python_koans#194]
The CI builds appear to be failing on Python 3.2 and 3.3. From what I saw it looks like the installations for those versions of Python might not be available any more at the URLs we're specifying. Not sure if we want to try to fix that or consider dropping support for those versions of Python.

See: https://travis-ci.org/gregmalcolm/python_koans/builds/544882513

**Comment by kjc** [comment]
Details:  The logs show (failed) attempts to download Python installers from:

* https://storage.googleapis.com/travis-ci-language-archives/python/binaries/ubuntu/16.04/x86_64/python-3.2.tar.bz2

* https://storage.googleapis.com/travis-ci-language-archives/python/binaries/ubuntu/16.04/x86_64/python-3.3.tar.bz2

Walking up the URL directory paths, the first non-error I got was from https://storage.googleapis.com/travis-ci-language-archives/ --- where a Ctrl-F search does not find the string "python" at all.

I'm not familiar with Travis (nor did I know Google apparently hosts old versions of Python), so I don't know where to go from here.

**Comment by kjc** [comment]
Also, we don't seem to have tests for Python 3.6 or 3.7.

**Comment by hugovk** [comment]
It's time to drop 3.2 and 3.3. Please see PR #207, and #208 to add 3.6 to 3.8.

**Comment by gregmalcolm** [comment]
As usual, I've come to the party late, but I just accepted a PR that removed Python 2 on the grounds that is no longer current. Also I figure we probably don't need to travis to build anything but the most current of Python 3s, so I changed it to the current latest, 3.9. I was tempted to tell travis to use tje "nightly build" version of python but I wimped. Maybe next time we get out of date we should just do it? :)

I'll go ahead and close up on this now.
